resourcegraph.tf

```
@@ -0,0 +1,84 @@
+resource "azurerm_automation_module" "az_accounts" {
+  name                    = "Az.Accounts"
+  resource_group_name     = var.resource_group.name
```

```suggestion
  resource_group_name     = var.automation_account.resource_group_name
```
Since this always needs to be the same RG as the automation account I would force this. Same for other resources like the runbook and schedule